alexthompson207/current-climate
# Test/favorites view

# PR CurrentClimate

### What does this do?


### Are there any known bugs?


### Summary of the changes
Add Testing, both happy and sad paths to the Favorites view

### What ticket(s) do the changes resolve?
Closer #29 
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
N/A

### How to Test Changes?
Run `npm run cypress` in this branch. All tests should pass for `favoritesView.js` and the other test files should pass as well.

### Next Iterations
- Add responsive styling to page.
